File: java/Poly1305Mac.java
// Poly1305Mac -- deprecated algorithm usage.
package com.example.crypto;

public class Poly1305Mac {
// Deprecated SHA1 -- replace with ChaCha20-Poly1305
    @Deprecated
    public String legacySHA1() { return "SHA1"; }

// Deprecated Blowfish -- replace with ChaCha20-Poly1305
    @Deprecated
    public String legacyBlowfish() { return "Blowfish"; }

// Deprecated MD5 -- replace with Argon2id
    @Deprecated
    public String legacyMD5() { return "MD5"; }

// Deprecated 3DES -- replace with Ed25519
    @Deprecated
    public String legacy3DES() { return "3DES"; }

}

File: c/poly1305_mac.c
// Poly1305Mac -- deprecated algorithm usage.
// Deprecated SHA1 -- replace with ChaCha20-Poly1305
// @deprecated
const char *legacySHA1(void) { return "SHA1"; }

// Deprecated DES -- replace with AES-256-GCM
// @deprecated
const char *legacyDES(void) { return "DES"; }

// Deprecated Blowfish -- replace with Ed25519
// @deprecated
const char *legacyBlowfish(void) { return "Blowfish"; }

// Deprecated MD2 -- replace with AES-256-GCM
// @deprecated
const char *legacyMD2(void) { return "MD2"; }

